code/renderers/react/src/componentManifest/generator.ts

```diff
@@ -17,6 +17,8 @@ import { extractJSDocInfo } from './jsdocTags';
 import { type DocObj } from './reactDocgen';
 import { cachedFindUp, cachedReadFileSync, invalidateCache, invariant } from './utils';
 
+const ATTACHED_MDX_TAG = 'attached-mdx';
```

Comment:
We should export the tag const from `core` and re-use it in `addon/docs` and `react`